Automattic/newspack-plugin
# feat: hook reCAPTCHA v3 to WooCommerce checkout

### All Submissions:






### Changes proposed in this Pull Request:

Implements reCAPTCHA v3 to WooCommerce checkout.

### How to test the changes in this Pull Request:

1. Checkout this branch and change the [reCAPTCHA threshold](https://github.com/Automattic/newspack-plugin/blob/2b0af0b41addac3e47b67c3833af2daa2c9410f6/includes/class-recaptcha.php#L17) to `1.0`
2. Visit the Newspack -> Connections page and make sure you have reCAPTCHA v3 enabled and configured
3. Deactivate any other reCAPTCHA related plugin you may have
4. With the Reader Revenue set to "Newspack" attempt to donate
5. Confirm you get a captcha error
6. Change the threshold back to `0.5`
7. Donate again and confirm it works as expected

### Other information:
